programs/gzip.c

```diff
@@ -576,6 +591,11 @@ tmain(int argc, tchar *argv[])
 			 *  option as a no-op.
 			 */
 			break;
+		case 't': /// test
```

Comment:
Please keep the options in alphabetical order.

Reply:
Sure